src/framework/audio/internal/synthesizers/fluidsynth/soundmapping.h

```diff
@@ -1000,7 +1000,10 @@ inline const ArticulationMapping& articulationSounds(const mpe::PlaybackSetupDat
     }
 
     if (setupData.category == mpe::SoundCategory::Winds) {
-        return WINDS;
+        soundId = setupData.soundId();
```

Comment:
This doesn't do anything

Reply:
it mostly certain does!

Reply:
what exactly?

Reply:
are you saying that at this point, soundId already has that value? When I looked that didn't seem to be true in all cases

Reply:
Yes, it has the correct value. We declare it here:
<img width="472" alt="Screenshot 2024-05-20 at 11 56 41" src="https://github.com/musescore/MuseScore/assets/67867444/525db4e7-0ae3-4cf6-a546-a89584f0aaed">